ecosystem_cicd_tools/release.py

```diff
@@ -247,3 +244,49 @@ def blueprint_release_with_latest(plugin_name,
         )
     latest_release = get_most_recent_release()
     update_latest_release_resources(latest_release)
+
+def plugin_release_with_latest(plugin_name,
+                               version=None,
+                               plugin_release_name=None,
+                               plugins=None):
+    # if we have release for this version we do not want update nothing
+    if not get_release(version):
```

Comment:
`plugin_release_with_latest` should also call create on the new release.

Reply:
line 267

Reply:
You mean 254?